modules/gdscript/gdscript_function.h

```diff
@@ -612,6 +619,7 @@ class GDScriptFunction {
 	Variant call(GDScriptInstance *p_instance, const Variant **p_args, int p_argcount, Callable::CallError &r_err, CallState *p_state = nullptr);
 
 #ifdef DEBUG_ENABLED
+	void _profile_native_call(uint64_t t_taken, String function_name, String instance_class_name);
```

Comment:
```suggestion
	void _profile_native_call(uint64_t p_t_taken, String p_function_name, String p_instance_class_name = String());
```
Default argument is defined here